### PR TITLE
InstData Import: pass description on to created instdasta

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/InstrumentData/Command/Import/Basic_bam.t [hunk 1]
```diff
@@ -51,6 +52,7 @@ my $instrument_data = $instrument_data[0];
 is($instrument_data->original_data_path, $source_bam, 'original_data_path correctly set');
 is($instrument_data->import_format, 'bam', 'import_format is bam');
 is($instrument_data->sequencing_platform, 'solexa', 'sequencing_platform correctly set');
+is($instrument_data->description, 'We took some DNA and sheared it and sequenced it.', 'description cprrectly set');
```
Comment (tmooney) [inline lib/perl/Genome/InstrumentData/Command/Import/Basic_bam.t]: The `o` in `correctly` has gained a tail!

**Comment by tmooney** [comment]
+1 :pig2:

**Comment by apregier** [comment]
+1 - another option would be not to expose description as a special parameter to the command and just let people set it via instrument_data_properties

**Comment by ebelter** [comment]
@apregier I would prefer that, too. Chip was very adamant about having this be a separate option as some of his scripts depend on it.

**Pull request merged**